docs/general/ad-filtering/create-own-filters.md

```diff
@@ -1417,6 +1417,8 @@ If a request matches multiple `$referrerpolicy` rules not disabled by exceptions
 * `@@||example.com^$referrerpolicy=unsafe-url` disables the previous rule.
 * `@@||example.com/abcd.html^$referrerpolicy` disables all `$referrerpolicy` rules on `example.com/abcd.html`.
 
+> `$referrerpolicy` rules are compatible only with `$document` and `$subdocument` [content type modifiers](#content-type-modifiers).
```

Comment:
usually it looks like
```suggestion
> **Compatibility with other modifiers**
>
> `$referrerpolicy` rules are compatible only with `$document` and `$subdocument` [content type modifiers](#content-type-modifiers).
```

Reply:
Another option is to place it under `restrictions`, but `Compatibility with other modifiers` looks like more common way :)